trytond/trytond/backend/postgresql/table.py

```diff
@@ -536,6 +536,29 @@ def set_indexes(self, indexes, concurrently=False):
                 cursor.execute(SQL('DROP INDEX {}').format(Identifier(name)))
         self.__indexes = None
 
+    def dump_indexes(self, indexes, file, concurrently=False):
+        def sql_quote(o):
+            if isinstance(o, str):
+                return f"'{o}'"
+            else:
+                return str(o)
+
+        for index in indexes:
+            translator = self.index_translator_for(index)
+            if translator:
+                name, query, params = translator.definition(index)
+                name = '_'.join([self.table_name, name])
+                name = ('idx_'
+                    + self.convert_name(name, reserved=len('idx_')))
+                file.write(
+                    ('CREATE INDEX {} IF NOT EXISTS {} ON {} USING {}\n'
```

Comment:
missin `;`.
Also, index / table names should be enclosed in double quotes to handle special characters (dashes) in names.